lib/LocalVariables/LocalVariableBuilder.cpp

```diff
@@ -21,6 +21,12 @@
 
 using namespace llvm;
 
+RegisterIRHelper StackFrameMarker("revng_stack_frame",
+                                  "absent in decompiled code");
```

Comment:
1. Why are you using "absent in decompiled code" as description?
2. Where do we use `IRHelper::Description`? Is it user facing?

Reply:
If I recall correctly, this started as a pad you made listing these and mentioning where they cease to exist. Hence, to not lose that information, a description was born.

As for why `absent in decompiled code` - it's because this exist in IR but is never emitted by the backend with this name - hence the "decompiled code" as the first point it disappears.

---------------------

That being said, I'm not attached to these. I'd rather we keep them over outright dropping them - as this way I never need to look for this information is I ever need it. But that's not a strong argument.